lib/commands/eval.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+    permissions: ["developer"],
+    guildOnly: true,
+    cooldown: 5000,
+    shortDescription: 'Evaluates a command',
+    longDescription: 'Evaluates arbitrary code. Only developers have access to this command.',
+    run: async function(client, m, args) {
+        let expression = args.join(" ");
+        try {
+            let result = eval(expression);
```

Comment:
For simplicity, this could have a few precalculated variables (namely: `message` (instead of `m`), `author`, `guild`, `channel`, `bot`, `database`

It could also be an async function instead of just an eval. This would allow using awaits in the eval'd code and more complex code for debugging.

Reply:
It's mostly about doing a shortcut to often used things. I'm not really against keeping it as it is, but I'd like if `m` was at least renamed to `message`.

Thoughts on the async function?

Reply:
Answered after discussion on Discord. Async functionality is moved to a new command, `exec`.